### ISSUE TITLE
[YCQL][Question]

### ISSUE THREAD

**Issue by zeinali-ali** [yugabyte/yugabyte-db#25416]
Jira Link: [DB-14648](https://yugabyte.atlassian.net/browse/DB-14648)
### Description

Hi does [auto_splitting](https://docs.yugabyte.com/preview/architecture/docdb-sharding/tablet-splitting/?_gl=1*oyc9ma*_gcl_au*MjMxNzQ4MzE3LjE3MzQzNTkyMTY.#enable-automatic-tablet-splitting) work for YCQL tables or we should consider growth of data when we gonna create tables?

we are using a cluster of version 2024.2.0.0-b145 on kubernetes

### Issue Type

kind/question

### Warning: Please confirm that this issue does not contain any sensitive information

- [X] I confirm this issue does not contain any sensitive information.

[DB-14648]: https://yugabyte.atlassian.net/browse/DB-14648?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

**Comment by ddorian** [comment]
Yes, it is supported. It's not supported when file based TTL is enabled https://github.com/yugabyte/yugabyte-db/issues/10837.
It works on normal per-row TTL though.
